vaults/test.yml

```diff
@@ -1,11 +1,11 @@
 ---
 cloud_token: 'this is the one from console.redhat.com'
-offline_token: 'this is the one linked below about api token'
+offline_token: 'this is the one linked in readme about api token'
 rh_username: 'redhat user login (this is used to attach your subs to controller)'
 rh_password: 'password for redhat account'
-root_machine_pass: 'password for root user on builder (if not root user more changes will need to be made)'
-ah_token_password: 'this will create and use this password can be generated'
-controller_api_user_pass: 'this will create and use this password can be generated'
+root_machine_pass: 'password for root user on builder (for non root user, other changes will be required)'
+ah_token_user_pass: 'this is for a new local account, whatever you put here will be that local account password'
+controller_api_user_pass: 'this is for a new local account, whatever you put here will be that local account password'
```

Comment:
the wording on these is just hard to understand. the first
If using a non root account, changes to the playbooks will be required. 

The 2nd, What is a local account to hub/controller, AFAIK you are referring to machine local accounts versus native user accounts on the application itself. 

Not sure what to call them, controller user accounts/hub user account, maybe?